pkg/alertmanager/multitenant_test.go

```diff
@@ -112,10 +113,36 @@ func TestLoadAllConfigs(t *testing.T) {
 	require.True(t, exists)
 	require.Equal(t, simpleConfigTwo, currentConfig.RawConfig)
 
-	// Test Delete User
+	// Test Delete User, ensure config is remove but alertmananger
+	// exists and is set to inactive
 	delete(mockStore.configs, "user3")
 	require.NoError(t, am.updateConfigs())
 	currentConfig, exists = am.cfgs["user3"]
 	require.False(t, exists)
-	require.Equal(t, alerts.AlertConfigDesc{}, currentConfig)
+	require.Equal(t, "", currentConfig.RawConfig)
+
+	userAM, exists := am.alertmanagers["user3"]
+	require.True(t, exists)
+	userAM.activeMtx.Lock()
+	require.False(t, userAM.active)
+	userAM.activeMtx.Unlock()
```

Comment:
I would suggest to add a `IsActive()` function to the alert manager, so that we can use it here and below.

Reply:
👍